auto_submit/lib/service/validation_service.dart

```diff
@@ -367,13 +367,8 @@ class ProcessMergeResult {
 typedef RetryHandler = Function();
 
 /// Runs the internal processMerge with retries.
-Future<void> _runProcessMergeWithRetries(RetryHandler retryHandler, {RetryOptions? retryOptions}) {
-  final RetryOptions r = retryOptions ??
-      const RetryOptions(
-        delayFactor: Duration(milliseconds: Config.backOfMultiplier),
-        maxDelay: Duration(seconds: Config.maxDelaySeconds),
-        maxAttempts: Config.backoffAttempts,
-      );
+Future<void> _runProcessMergeWithRetries(RetryHandler retryHandler, RetryOptions retryOptions) {
+  final RetryOptions r = retryOptions;
```

Comment:
Nit: no need to define an extra variable.

Reply:
Fixed.